ServiceWebsite/ServiceWebsite.AcceptanceTests/Helpers/TestLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using ServiceWebsite.Common;
+
+namespace ServiceWebsite.AcceptanceTests.Helpers
+{
+    public class TestLogger
+    {
+        const string CATEGORY = "Acceptance Test";
+        public static void Log(string message)
+        {
+            ApplicationLogger.Trace(CATEGORY,
```

Comment:
Logging to AppInsights for your tests might not be the best way to go about this. Since we know we use NUnit as our test framework it would be better suited to use the '[TestContext's](https://github.com/nunit/docs/wiki/TestContext) TextWriter. That way any errors are written to the console for each test and it's quicker to find than to log into AppInsights and filter out. Also, since the pipeline will run the tests, we can see the errors in the logger on the failed test immediately making it quicker to find the source of the issue

Reply:
Ok let's discuss this approach.